app/views/journeys/capitalallowances/balancingAllowance/BalancingAllowanceAmountView.scala.html

```diff
@@ -0,0 +1,59 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.journeys.capitalallowances.balancingAllowance.routes
+@import models.common.{BusinessId, TaxYear, UserType}
+@import viewmodels.InputWidth._
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode, userType: UserType, taxYear: TaxYear, businessId: BusinessId)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages(s"amountSpentOnEvcp.title.$userType"))) {
+
+    @if(form.errors.nonEmpty) {
+        @govukErrorSummary(ErrorSummaryViewModel(form))
+    }
+
+    @formHelper(action = routes.BalancingAllowanceAmountController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
+
+        @govukInput(
+            InputViewModel(
+                field = form("value"),
+                label = LabelViewModel(HtmlContent(
+                    s"""
+                    <div>
+                        <label class="govuk-label govuk-label--l">
+                            ${messages(s"balancingAllowanceAmount.title.$userType")}
+                        </label>
+                    </div>
+                    """
+                ))
```

Comment:
This can just be
 ```
label = LabelViewModel(messages(s"balancingAllowanceAmount.title.$userType"))
                .asPageHeading(size = Large)
```
with import 
@import viewmodels.LabelSize.Large